src/routes/stats-router.js

```diff
@@ -0,0 +1,40 @@
+const { Router } = require("express");
+
+const router = new Router();
+
+router.get("/", (request, response) => {
+  const { db } = response.locals;
```

Comment:
Getting the database instance from the `response.locals` that was injected with the `attachDatabaseMiddleware`.